### PR TITLE
branch initial stup

### PR DESCRIPTION
**Pull request by aramiiian**

### CODE REVIEW

**Review by m-avagyan — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -24,7 +24,7 @@ header {
 header .logo {
     width: 147px;
     height: 30px;
-    background: url(logo/logo2.png) no-repeat center / 100%;
+    background: url(img/logo2.png) no-repeat center / 100%;
```
Comment (m-avagyan) [inline style.css]: ete du es nkarnery img folderi mej es pahum, aveli lav a rename anes et foldery `images`

File: style.css [hunk 2]
```diff
@@ -305,12 +305,159 @@ header .logo {
...
+    padding: 100px 0 40px 0;
+}
+.work .title-work .work-nav {
+    text-align: center;
+    padding: 0 0 40px 0;
+}
+.work .title-work .work-nav a {
+    color: #7c7c7c;
+    text-decoration: none;
+    font: 16px 'Montserrat', sans-serif;
```
Comment (m-avagyan) [inline style.css]: ete sax fontert `Montserrat` taratesakov en, aveli lav a verevy senc ban gres
```
* {
   font-family: 'Montserrat'
}
```

esi nshanakum a vor sax elementnerit font familyn dranic sarqi

File: style.css [hunk 3]
```diff
@@ -305,12 +305,159 @@ header .logo {
...
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    gap: 20px;
+    
+}
+
+.work .portfilo .img1 {
+    background: url(img/pf1.gif) no-repeat center  / cover;
```
Comment (m-avagyan) [inline style.css]: du vor heto kardas pf1.gif haskanalu es et inchi masin a?

File: style.css [hunk 4]
```diff
@@ -305,12 +305,159 @@ header .logo {
...
+    border: 1px solid;
+    width: 350px;
+    height: 279px;
+}
+.work .portfilo .img4 {
+    background: url(img/pf4.jpg) no-repeat center  / cover;
+    border: 1px solid;
+    width: 350px;
+    height: 279px;
+    margin-top: -72px;
```
Comment (m-avagyan) [inline style.css:365]: minusov marginnery lav practica chen

File: style.css [hunk 5]
```diff
@@ -305,12 +305,159 @@ header .logo {
...
+    width: 50%;
+    margin-left: 25%;
+    
+}
+.team .team-title h2 {
+    color: #fff;
+    font: bold 60px 'Montserrat', sans-serif;
+}
+.team .team-title p {
+    color: #9E9E9E;
```
Comment (m-avagyan) [inline style.css]: ashxati sax guynert mianman grel

xosqi mi tex greles rgb, mi tex hex, mi tex el hex mecatarov

File: style.css [hunk 6]
```diff
@@ -305,12 +305,159 @@ header .logo {
...
+    width: 25%;
+    height: 700px;
+    background: url(img/tm1.png) no-repeat center / cover;
+    border: 1px solid #Fff; 
+}
+.team .team-img .tm2 {
+    width: 25%;
+    height: 700px;
+    background: url(img/tm2.png) no-repeat center / cover;
+    border: 1px solid #Fff;
```
Comment (m-avagyan) [inline style.css]: #fff gone

File: style.css (hunk 1 above)
Comment (aramiiian) [reply style.css]: poxe push em are

Comment (m-avagyan) [reply style.css]: ches are normal nayi

File: style.css (hunk 2 above)
Comment (aramiiian) [reply style.css]: poxelem

**Review by m-avagyan — APPROVED** [review]

File: style.css (hunk 3 above)
Comment (aramiiian) [reply style.css]: es ha mnacacy chgitem bayc poxe pushelem

File: style.css (hunk 4 above)
Comment (aramiiian) [reply style.css:365]: -

File: style.css (hunk 5 above)
Comment (aramiiian) [reply style.css]: +

File: style.css (hunk 6 above)
Comment (aramiiian) [reply style.css]: +

**Review by m-avagyan — APPROVED** [review]

**Pull request merged**